cico_setup.sh

```diff
@@ -102,26 +102,35 @@ function run_tests_with_coverage() {
 }
 
 function tag_push() {
-  TARGET=$1
-  docker tag fabric8-auth-deploy $TARGET
-  docker push $TARGET
+  local tag
```

Comment:
why not `local tag=$1` ?

Reply:
it should have been like that, sorry I didn't see it before

Reply:
@jmelis no worries. I'm not even using `local` yet. Will implement it in the future in our shell scripts. ;)